src/common/api/worker/EventQueue.ts

```diff
@@ -59,10 +59,11 @@ export function batchMod(batchId: Id, batch: ReadonlyArray<EntityUpdate>, entity
 type LastOperationKey = string & { __brand: "lastOpeKey" }
 
 function lastOperationKey(update: EntityUpdate): LastOperationKey {
+	const typeIdentifier = `${update.application}/${update.type}`
```

Comment:
we could probably avoid allocating another string if we just inlined this in both cases but maybe that will get optimized this way anyway

Reply:
Is this just a concern about performance (memory allocation)? My rationale to avoid having duplicate code/strings. Not sure which one is better. I will just merge this now.